src/util/commandline/command.ts

```diff
@@ -137,7 +137,12 @@ export class Command {
         return Promise.resolve(Result.failure(Result.ErrorCodes.InvalidParameter, `Unknown output format ${this.format}`));
       }
     }
-    this.clientFactory = createAppCenterClient(this.command, await telemetryIsEnabled(this.disableTelemetry));
+
+    this.clientFactory = createAppCenterClient(
+      this.command,
+      await telemetryIsEnabled(this.disableTelemetry || this.command[0] === "telemetry")
```

Comment:
If I understand correctly this will always disable telemetry on that command, even if telemetry was previously enabled (i.e. false vs. undefined). It probably does not really matter but I'd like to make sure that was the intent.

Reply:
It will not emit telemetry when calling this command, regardless of the effect of the command itself (enabling or disabling telemetry for other commands). Alternative solution is to put `disableTelemetry = true` inside the constructors of the `telemetry` `on` and `off` commands.

Reply:
And that is the intent. If you're performing an operation enabling or disabling telemetry, you don't then during that want to be asked a question about it, performing the operation twice and confusing you.

Reply:
I think this approach is fine. It's simple.

Reply:
I like the other solution a bit better since it doesn't mess with the main `Command` class so I'll update the PR to it. Same effect :)